mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/kafka/producers/TiltakstypeKafkaProducer.kt

```diff
@@ -19,7 +19,9 @@ class TiltakstypeKafkaProducer(
         val record: ProducerRecord<String, String?> = ProducerRecord(
             config.topic,
             value.id.toString(),
-            Json.encodeToString(value),
+            Json.encodeToString(
+                if (value.tiltakskode != null) value else null,
```

Comment:
Hva skjer her? Setter melding til  `null` for alle tiltak som ikke er gruppetiltak? Kunne vi heller gjort en sjekk i `ArenaAdapterService` og kun kjørt en `publish` på gruppetiltakene der i stedet?

Reply:
Har vi forresten fått bekreftet OK at vi kan fjerne individulle tiltak fra topic? :sweat_smile:

Reply:
Ja mener vi diskuterte å sende tombstone på de andre. Og ja har fått bekreftet det OK fra komet som er eneste konsument, eller har ikke spurt dvh, kan si ifra til dem også